apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -1678,49 +1713,53 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
-                        }
-                        type="radio"
-                        children={ getAllowedListForAccessToken(metadata.accessTokenBindingType, true) }
-                        readOnly={ readOnly }
-                        data-testid={ `${ testId }-access-token-type-radio-group` }
-                        listen={ (values: Map<string, FormValue>) => {
-                            setIsTokenBindingTypeSelected(
-                                values.get("bindingType") !== SupportedAccessTokenBindingTypes.NONE
-                            );
-                        } }
-                    />
-                    <Hint>
-                        <Trans
-                            values={ { productName: config.ui.productName } }
-                            i18nKey={
-                                "console:develop.features.applications.forms.inboundOIDC.sections" +
-                                ".accessToken.fields.bindingType.description"
-                            }
-                        >
-                            Select type <Code withBackground>SSO-session</Code> to allow productName to bind the
-                            <Code withBackground>access_token</Code>
-                            and the
-                            <Code withBackground>refresh_token</Code>
-                            to the login session and issue a new token per session. When the application
-                            session ends, the tokens will also be revoked.
-                        </Trans>
-                    </Hint>
-                </Grid.Column>
-            </Grid.Row>
+            {
+                !isM2MApplication && (
+                    <Grid.Row columns={ 1 }>
+                        <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                            <Field
+                                ref={ bindingType }
+                                label={
+                                    t("console:develop.features.applications.forms.inboundOIDC.sections" +
+                                        ".accessToken.fields.bindingType.label")
+                                }
+                                name="bindingType"
+                                default={
+                                    initialValues?.accessToken?.bindingType
+                                        ? initialValues.accessToken.bindingType
+                                        : metadata?.accessTokenBindingType?.defaultValue
+                                        ?? SupportedAccessTokenBindingTypes.NONE
+                                }
+                                type="radio"
+                                children={ getAllowedListForAccessToken(metadata.accessTokenBindingType, true) }
+                                readOnly={ readOnly }
+                                data-testid={ `${ testId }-access-token-type-radio-group` }
```

Comment:
Let's use `data-componentid` instead

Reply:
data-testid was used to preserve the compatibility with tests